com.wetgenes.gamecake.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: com.wetgenes.gamecake
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gamecake
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+modules:
+  - name: luajit
```

Comment:
maybe you should use the shared-module

https://github.com/flathub/shared-modules/blob/master/luajit/luajit.json

Reply:
maybe, I figured using modules felt less explicit.

Sometimes I've hit rare luajit issues so would like to know and control exactly which version of luajit is in use, just in case.